src/main/java/org/eclipse/yasson/internal/deserializer/DeserializationModelCreator.java

```diff
@@ -298,9 +296,10 @@ private ModelDeserializer<JsonParser> createMapDeserializer(CachedItem cachedIte
                                                                    ClassCustomization.empty(),
                                                                    JustReturn.instance(),
                                                                    MAP_KEY_EVENTS);
+        ClassModel valueClassModel = jsonbContext.getMappingContext().getOrCreateClassModel(ReflectionUtils.getRawType(valueType));
```

Comment:
If `valueType` is a `WildcardType` (?), using `getRawType(valueType)` is resulting in a jakarta.json.bind.JsonbException: Error resolving runtime type for type: ?

When using `resolveRawType(chain, valueType)`, a `WildcardType` is deserialized correctly.

Reply:
I'll double check, but what you're saying does seem correct. Nice catch!